sdk/azidentity/managed_identity_client.go

```diff
@@ -293,7 +293,12 @@ func (c *managedIdentityClient) createServiceFabricAuthRequest(ctx context.Conte
 	q.Add("api-version", serviceFabricAPIVersion)
 	q.Add("resource", strings.Join(scopes, " "))
 	if id != nil {
-		q.Add(qpClientID, id.String())
+		log.Write(EventAuthentication, "Service Fabric doesn't support selecting a user-assigned identity at runtime")
```

Comment:
Nit: This log message is a bit different than the others, is that intentional?

Reply:
Also, should these messages have some kind of prefix like `WARNING: ` so they stand out a bit more?

Reply:
Service Fabric does support user-assigned identities, but you configure one in the application manifest and can't select a different one at runtime as you can on App Service, for example. Arc and Cloud Shell don't support user-assigned identity at all.